ci/prow/lint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
I would not add the `.sh` extension to those files. Apart from that, LGTM

Reply:
Why not?

Reply:
The practice of no extension is for all executables, not just shell scripts. See the [Google style guide](https://google.github.io/styleguide/shellguide.html#file-extensions).

Reply:
Have commented on the Prow PR - https://github.com/kubernetes/test-infra/pull/27012

Reply:
Fixed.